Python/Product/PythonTools/ptvsd_launcher.py

```diff
@@ -71,8 +71,8 @@
 # Load the debugger package
 try:
     if bundled_ptvsd:
-        ptvs_lib_path = os.path.dirname(__file__)
-        sys.path.insert(0, ptvs_lib_path)
+        ptvs_lib_path = os.path.join(os.path.dirname(__file__), 'Packages')
+        sys.path.append(ptvs_lib_path)
```

Comment:
Is it still possible to use the old debugger?

Reply:
Um... yeah... good point. I'll add another option to the launch script for that so we can still do it.